### DISCUSSION TITLE
关于论文或书籍写作，有没有什么best practice介绍的文字？

### DISCUSSION THREAD

**Discussion by tuxdio** [rbind/yihui#1554] [Q&A] [answered]
想翻译一本书，觉得用Word或者LaTex都太重了，想用web/markdown编写，但不知道是直接用第三方平台还是自己在电脑上搭建一个平台。不知道谢博的写作工作流是如何管理的？

**Comment by tuxdio** [comment]
谢博是不是直接在bookdown.org上写作？
https://yihui.org/en/2018/08/bookdown-crc/
https://yihui.org/en/2018/07/whole-bookdown/

**Reply by yihui** [reply, marked as answer]
bookdown.org 上的书都是在本地写好了发布上去的，它没有在线编辑平台。工具是次要的，或者说取决于你需要什么功能。一本书无非就是一系列 Rmd 文件，每个文件是一章。重点是把内容憋出来，工具、工作流都不太重要。

bookdown 的功能基本上都已经在 Quarto 中重写了，你可以直接上手 Quarto。

**Reply by tuxdio** [reply]
3ks!
